src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { fetchWithVouchRedirect } from './fetchWithVouchRedirect';
+export { AuthPopup } from './AuthPopup';
```

Comment:
does this need to be exported?

Reply:
AuthPopup has to be exported to be able to be mounted in the backstage App router

Reply:
ah ok got it.